lib/enkf/ensemble_config.cpp

```diff
@@ -884,6 +915,12 @@ enkf_config_node_type * ensemble_config_add_gen_kw( ensemble_config_type * confi
   return config_node;
 }
 
+//enkf_config_node_type * ensemble_config_add_gen_param( ensemble_config_type * config , const char * key , bool forward_init) {
```

Comment:
Easter egg legacy code?

Reply:
Yes, I meant this exactly - low hanging fruits :)